tests/system-tests/rdscore/tests/00_validate_top_level.go

```diff
@@ -26,7 +26,7 @@ var _ = Describe(
 	Label("rds-core-workflow"), func() {
 		Context("Configured Cluster", Label("clean-cluster"), func() {
 			It("Verifies SR-IOV workloads on same node and different networks",
-				Label("sriov", "sriov-same-node-different-nets"),
+				Label("sriov", "debug"),
```

Comment:
don't forget to restore when it's ready 😸

Reply:
sure, sorry, leftovers